dumper/lib-ext-hive-metastore/src/main/java/com/google/edwmigration/dumper/ext/hive/metastore/HiveMetastoreThriftClient_v2_3_6.java

```diff
@@ -123,31 +124,42 @@ public String getLocation() {
             @Nonnull
             @Override
             public List<? extends Field> getFields() throws Exception {
+                if (table.getSd() != null) {
+                    List<com.google.edwmigration.dumper.ext.hive.metastore.thrift.api.v2_3_6.FieldSchema> cols = table.getSd().getCols();
+                    if (cols != null) {
+                        return cols.stream().map(this::toField).collect(Collectors.toList());
+                    }
+                }
```

Comment:
I'd love a comment saying why this condition is here, and why there are two paths.

Reply:
Done.